htdocs/core/class/html.formfile.class.php

```diff
@@ -1320,5 +1328,34 @@ public function listOfLinks($object, $permtodelete=1, $action=null, $selected=nu
         return $nboflinks;
     }
 
+    /**
+     * Show detail icon with link for preview
+     * @param array     $file   File
+     * @param string    $modulepart     propal, facture, facture_fourn, ...
+     * @param string    $relativepath   Relative path of docs
+     * @return string   $out            Output string with HTML
+     */
+    function showPreview($file, $modulepart, $relativepath){
+        global $langs;
+
+        $out='';
```

Comment:
Added, but if (empty($conf->dol_use_javascript)) return '';  no work correctly for me

I have used;
  if (empty($conf->use_javascript_ajax)) return '';

Is it correct?